tempest/src/main/kotlin/app/cash/tempest/internal/DynamoDbQueryable.kt

```diff
@@ -66,7 +66,11 @@ internal class DynamoDbQueryable<K : Any, I : Any>(
     }
     if (specificAttributeNames.isNotEmpty()) {
       query.withSelect(SPECIFIC_ATTRIBUTES)
-      query.projectionExpression = specificAttributeNames.joinToString(", ")
+      val specificAttributeNamesAliased = specificAttributeNames.map {
+        reservedWordsMapping.getOrDefault(it, it)
+      };
+      query.projectionExpression = specificAttributeNamesAliased.joinToString(", ")
+      query.expressionAttributeNames = reservedWordsAliasMapping;
```

Comment:
We should filter this map down to only what we need?

Reply:
Yeah there were actually tests that required that. Updated.